index.html

```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
                 </div>
             </div>
         </section>
+        <section class = "sect_content">
+            <div class = "container">
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Timer.png" alt="timer" class="feature__icon">

```

Comment:
1) For small icons it's better to embed them inline as base64 to make fewer requests to the server
2) Use [TinyPNG](https://tinypng.com/) to compress your icons, 
I've tried it with dashboard items, the result is 2 times smaller `1.4kb -> 711b` but looks the same
3) [Optional] consider `alt="timer icon"`